spec/web-ui/README.md

```diff
@@ -1,2 +1,34 @@
 ## Web-ui
```

Comment:
@Gayatri212 please adding docker build badge on the top of the readme. as follows-

```![Docker Cloud Build Status](https://img.shields.io/docker/cloud/build/enterpriseconnect/webui)```